dev-tools/travis/travis-script.sh

```diff
@@ -31,7 +31,14 @@ else
   # Travis only has 3GB of memory, lets use 1GB for build, and 1.5GB for forked JVMs
   export MAVEN_OPTS="-Xmx1024m"
   
-  mvn --batch-mode test -fae -Pnative,all-tests '-P!include-shaded-deps' -Prat -pl "$2"
+  TEST_MODULES=$2
+  
+  if [ "$TRAVIS_JDK_VERSION" == "oraclejdk10" ] && [ "$TEST_MODULES" == "'!storm-client,!storm-server,!storm-core'" ]
+  then 
+    TEST_MODULES = '!storm-client,!storm-server,!storm-core,!external/storm-cassandra,!external/storm-hive'
```

Comment:
We have overloaded TEST_MODULES from being a pass-through to maven already for integration tests, lets just bite the bullet and turn it into an enum instead, so it is not as unexpected.

Reply:
Could you give an example what this would look like? Googling "travis enum" didn't pop up anything.

Reply:
Sorry I wasn't clear enough.  In `.travis.yaml` we have a set of MODULES that we build in parallel.  Currently these are passed unchanged to this script.  For most of them they are maven modules that just get passed directly to mvn, but in the case of INTEGRATION-TEST it has nothing to do with the maven modules and now with this change we are actually modifying the modules.  Because of this I would prefer to see us put in a layer of abstraction so `.travis.yaml` looks more like

```
env:
  - MODULES=CLIENT
  - MODULES=SERVER
  - MODULES=CORE
  - MODULES=EXTERNAL
  - MODULES=INTEGRATION-TEST
```

where the mapping is totally in travis-script.sh